Airwallex/Card/Internal/AWXUIContext+Card.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AWXUIContext+Card.swift
+//  Card
+//
+//  Created by Tony He (CTR) on 2024/7/26.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+@objc public extension AWXUIContext {
+    /**
+     Present the payment flow from card info collection view.
+     */
+
+    func presentCardPaymentFlowFrom(_ hostViewController: UIViewController, cardSchemes: [AWXCardBrand] = [AWXCardBrand.unknown, .visa, .amex, .mastercard, .discover, .JCB, .dinersClub, .unionPay]) {
```

Comment:
This card brand list has been reused a lot, what about creating a Constant to host the single source.